### DISCUSSION TITLE
General Relativity, blessing or curse?

### DISCUSSION THREAD

**Discussion by HanDeBruijn** [a-cosmology-group/acg#78] [Cosmology Models]
The subject is triggered by the article titled [Cosmology in Minkowski space](https://iopscience.iop.org/article/10.1088/1361-6382/acdb41) by [Lucas Lombriser](https://cosmology.unige.ch/users/lucas-lombriser) which has been posted in [Mathematicists find new glasses](https://github.com/orgs/a-cosmology-group/discussions/63).
1. The geometries in Einstein's General Relativity theory have been made "possible" by the discovery of [Non-Euclidean geometry](https://en.wikipedia.org/wiki/Non-Euclidean_geometry#Discovery_of_non-Euclidean_geometry) in the beginning of the 19th century. This has lead to the (mis)understanding that non-Euclidean geometries might be relevant to physics. However, the the only thing it shows is that Euclid's Fifth's Postulate - the one of the parallel lines - cannot be deduced from the other postulates; it's an independent axiom. Which as such is an important result. But it does not prove that physical space can be other than __Euclidean__. Non-Euclidean geometries can always be embedded in __common geometry__ - there are many such [Models of non-Euclidean geometry](https://en.wikipedia.org/wiki/Non-Euclidean_geometry#Models) in mathematics.
2. In Einstein's General Relativity theory, a four-dimensional space-time [__Manifold__](https://en.wikipedia.org/wiki/Manifold) is employed, which is __curved into nothingness__. The latter may be regarded as a distortion. A surface may be bent in a space with a higher dimension. But nowhere in Solid Mechanics - or elsewhere in classical physics - there exist [Applications of Manifolds not embedded in Euclidean Space](https://math.stackexchange.com/questions/3590072/applications-of-manifolds-not-embedded-in-euclidean-space).
3. In the following posting at Mathematics Stack Exchange it is argued that Euclid's Fifth's Postulate is not only essential for Euclidean geometry, but it's also essential for elementary algebra: [Non-Euclidean Geometrical Algebra for Real times Real?](https://math.stackexchange.com/questions/1951863/non-euclidean-geometrical-algebra-for-real-times-real). I have yet to see any decent example contradicting this evidence: _without the axiom of parallel lines elementary algebra cannot even exist_. However, Einstein's (tensor) calculus eventually rests on elementary algebra. How then can space-time of General Relativity be non-Euclidean?
4. A quote from [_Spacetime- Right or Wrong?](http://www.conspiracyoflight.com/Spacetime.html) by Doug Marett. _What space-time implies is that all events, past, present and future, are already written in the fabric of space. Such a conclusion relegates all human activity to simply being automatons acting out the frames of a preordained film strip. If we were to believe Einstein we would have to believe that there is no free will at all. That in and of itself should be reason enough to reject the space-time concept without further consideration!_
5. In [Léon Brillouin](https://en.wikipedia.org/wiki/L%C3%A9on_Brillouin)'s book [Relativity Reexamined](http://www.alternatievewiskunde.nl/einstein/relativity-reexamined_compress.pdf) we read on page 10 about the Theory of General Relativity: _a splendid piece of mathematics built on quicksand and leading to more and more mathematics about cosmology (a typical [__science-fiction__](https://en.wikipedia.org/wiki/Chronology_of_the_universe) process)_. And on page 31 it reads: _Einstein introduced a very heavy mathematical structure that goes much beyond any physical need._ A great deal of Big Bang cosmology - if not everything - is formulated in terms of this overly complicated (GR) mathematics.

What would be the alternative? I think that a Static Euclidean Universe (SEU), much like the one as proposed by [Eric Lerner](https://en.wikipedia.org/wiki/Eric_Lerner#The_Big_Bang_Never_Happened) comes close to the one that shall be adopted. No [Flatness problem](https://en.wikipedia.org/wiki/Flatness_problem), no [Horizon problem](https://en.wikipedia.org/wiki/Horizon_problem), no [Magnetic monopole](https://en.wikipedia.org/wiki/Magnetic_monopole) problem, no nothing of the kind. Just good old physics (and mathematics) as it should be.

**Comment by OlliSant** [comment]
4D is possibly good model too. With Q as diameter instead of infinity. Q is the longest possible distance in the reality.

On the previous thread Savorinen had two erroneous  things, he thinks that the space is absolutely infinite. It is not. And he tries to take off the gravity totally by a pushing force. Impossible.
